pithos/application.py

```diff
@@ -192,6 +189,7 @@ def do_activate(self):
         self.window.present()
 
     def do_shutdown(self):
+        self.withdraw_notification(self.get_application_id())
```

Comment:
Move this to the plugin.

Reply:
How? by connecting to the shutdown signal?

The docs for the shutdown signal say:

"The ::shutdown signal is emitted only on the registered primary instance immediately after the main loop terminates"

I'm not sure how we're suppose to withdraw an notification with no mainloop?

Reply:
That is odd phrasing that indeed sounds wrong.